ui/webui/test/check-storage

```diff
@@ -408,12 +419,8 @@ class TestStorageMountPoints(anacondalib.VirtInstallMachineCase):
         disk = "/dev/vda"
         dev = "vda"
         btrfsname = "btrfstest"
+        self.partition_disk(disk, [["--new=0:0:+1MiB", "--typecode=0:ef02"], "--new=0:0:+1GiB", "--new=0:0:+10GiB", "--new=0:0:0"])
```

Comment:
I would prefer to have it a bit more user friendly with the partitiion disk method taking parameters the partition sizes and the format types. So something like:
`partition_disk(disk,  [("biosboot", "1MiB"), ("ext4", "1GB"), ("btrfs", "10GB"), ("xfs", "")])`

Empty last size means use the rest. What do you think? I think that's more readable.

Reply:
Ok!, I'll look into it